Sources/TokamakCore/Views/Button.swift

```diff
@@ -45,6 +44,12 @@ extension Button: ParentView {
   }
 }
 
-public func buttonLabel(_ button: Button<Text>) -> String {
-  button.label.content
+/// This is a helper class that works around absence of "package private" access control in Swift
+public struct _ButtonProxy {
+  public let subject: Button<Text>
+
+  public init(_ subject: Button<Text>) { self.subject = subject }
+
+  public var label: _TextProxy { _TextProxy(subject.label) }
+  public var action: () -> () { subject.action }
```

Comment:
ok this makes a lot more sense! So it allows anyone with access to `_ButtonProxy` to read private properties of a `Button`.

Reply:
Yes, same as stand-alone functions did, but now grouped in a proxy wrapper type.